src/main/java/com/prgms/vouchermanager/util/io/ConsoleInput.java

```diff
@@ -0,0 +1,162 @@
+package com.prgms.vouchermanager.util.io;
+
+
+import com.prgms.vouchermanager.contorller.front.FrontController;
+import com.prgms.vouchermanager.domain.voucher.FixedAmountVoucher;
+import com.prgms.vouchermanager.domain.voucher.PercentDiscountVoucher;
+import com.prgms.vouchermanager.domain.voucher.Voucher;
+import com.prgms.vouchermanager.validation.InputValidation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import java.util.Scanner;
+import java.util.UUID;
+
+import static com.prgms.vouchermanager.domain.voucher.VoucherType.*;
+import static com.prgms.vouchermanager.exception.ExceptionType.*;
+import static com.prgms.vouchermanager.exception.ExceptionType.INVALID_VOUCHER_INFO;
+
+@Component
+public class ConsoleInput {
+    private final Scanner scanner;
+    private final InputValidation inputValidation;
+    private final ConsoleOutput consoleOutput;
+
+    private final static Logger logger = LoggerFactory.getLogger(FrontController.class);
+
+
+    public ConsoleInput() {
+        this.inputValidation = new InputValidation();
+        this.consoleOutput = new ConsoleOutput();
+        scanner = new Scanner(System.in);
+    }
+
+    public int inputFrontMenu() {
+
+        consoleOutput.printFrontMenu();
+
+        int menu ;
+
+        try {
+            menu = Integer.parseInt(scanner.next());
```

Comment:
```suggestion
            menu = scanner.nextInt();
```

nextInt() 메서드도 있는거 같은데 next()로 입력받아 Integer.parseInt로 변환하고 있는지 알 수 있을까요?

Reply:
값을 검증한다는데 꽂혀서 nextInt()를생각을 못했습니다. 😢